account_check_printing_report_base/data/report_paperformat.xml

```diff
@@ -15,4 +15,20 @@
         <field name="header_spacing">3</field>
         <field name="dpi">80</field>
     </record>
+
+    <record id="paperformat_euro_no_margin" model="report.paperformat">
+        <field name="name">Promissory Note Letter</field>
+        <field name="default" eval="False" />
+        <field name="format">custom</field>
+        <field name="page_height">297</field>
+        <field name="page_width">210</field>
+        <field name="orientation">Portrait</field>
+        <field name="margin_top">40</field>
+        <field name="margin_bottom">0</field>
+        <field name="margin_left">0</field>
+        <field name="margin_right">0</field>
+        <field name="header_line" eval="False" />
+        <field name="header_spacing">40</field>
+        <field name="dpi">90</field>
+    </record>
```

Comment:
It is created but never used, is that correct?

Reply:
Yes, it is correct, but it will be used on other 3 modules and it is added here for avoid repeated code.